src/openMVG/calibration/calibration.hpp

```diff
@@ -9,13 +9,13 @@ namespace calibration{
 /**
  * @brief This function computes the average of the reprojection errors.
  *
- * @param[in] objectPoints
- * @param[in] imagePoints
- * @param[in] rvecs
- * @param[in] tvecs
- * @param[in] cameraMatrix
- * @param[in] distCoeffs
- * @param[out] perViewErrors
+ * @param[in] objectPoints Coordinates of the 3D points in each image of the sequence.
```

Comment:
Coordinates of the 3D points associated to the detected points in each image of the sequence.